docs/kits/Business Partner Kit/Documentation BPDM/page_use-cases.md

```diff
@@ -0,0 +1,2294 @@
+---
+id: Use Cases
+title: Use Cases
+description: ''
+sidebar_position: 3
+---
+
+Here we have provided detailed examples of how the BPDM services can be used. This includes step-by-step instructions for each use case and code snippets showing how to make the API calls for better performance.
+
+## Use Case 1. Request Legal Entity by DE_VAT (via Legal Entity Controller)
```

Comment:
DE_VAT is concrete example technical key for the German VAT identification number as you mentioned in the section. Make this more a general use case description like "Request Legal Entity by Identifier Number" or similar.